modules/account-lib/src/coin/dot/unstakeBuilder.ts

```diff
@@ -0,0 +1,104 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import { UnsignedTransaction } from '@substrate/txwrapper-core';
+import { decode, methods } from '@substrate/txwrapper-polkadot';
+import BigNumber from 'bignumber.js';
+import { TransactionType } from '../baseCoin';
+import { InvalidTransactionError } from '../baseCoin/errors';
+import { MethodNames, UnstakeArgs } from './iface';
+import { Transaction } from './transaction';
+import { TransactionBuilder } from './transactionBuilder';
+import { UnstakeTransactionSchema } from './txnSchema';
+
+export class UnstakeBuilder extends TransactionBuilder {
```

Comment:
Can this be part of the Staking builder but just a different `TransactionType.StakingUnlock`?

Reply:
Separate builder makes it clearer in my opinion. Having to TransactionTypes wouldn't make sense